packages/design-picker/src/components/design-picker-category-filter/style.scss

```diff
@@ -1,3 +1,47 @@
+@import '@automattic/onboarding/styles/mixins';
+@import '@automattic/onboarding/styles/variables';
+@import '@automattic/typography/styles/fonts';
+
 .design-picker-category-filter {
 	flex: 1;
 }
+
+.design-picker-category-filter__sidebar {
+	display: none;
+
+	@include break-small {
+		display: block;
+	}
+}
+
+.design-picker-category-filter__dropdown {
+	@include break-small {
+		display: none;
+	}
+
+	font-family: $sans;
+	font-size: $font-body-small;
+	line-height: 24px;
+	font-weight: 500; // stylelint-disable-line scales/font-weights
+	border-radius: 4px; // stylelint-disable-line scales/radii
+	border-color: #c3c4c7;
+	padding: 0 18px 0 10px;
```

Comment:
NIT: maybe we should increase padding-right to avoid overlapping between text and background image.

Reply:
I thought I'd remembered that but obviously not, thanks for catching.